code/tests/functional/backend_api/resources/README.md

```diff
@@ -0,0 +1,9 @@
+# Why is this here?
+
+The file [9b5ad71b2ce5302211f9c61530b329a4922fc6a4](9b5ad71b2ce5302211f9c61530b329a4922fc6a4) is required to stop the 
```

Comment:
It depends entirely on the [tiktoken ](https://pypi.org/project/tiktoken/) library which is used by openai. They have an issue open for it [here](https://github.com/openai/tiktoken/issues/232) so if that gets resolved hopefully this will go away entirely.

Reply:
Got it thanks, maybe we should add the link to the issue to this README file so that the next person can check if it can be removed

Reply:
Good point, I've added that now!